### ISSUE TITLE
the center loss 

### ISSUE THREAD

**Issue by rabahMoulai** [pytorch/pytorch#68686]
the center loss result is vector or scalar ?

**Comment by jbschlosser** [comment]
Hey @rabahMoulai, I'm not quite sure what you are referring to - would you mind providing a bit more detail?

**Comment by rabahMoulai** [comment]
Hi @jbschlosser, thank you for your help, I'm trying to implement the loss function, published in the article "A Discriminative Feature Learning Approach for Deep Face Recognition", function (2) in the article. My problem is when subtracting from the instance x the center c we find a vector of the same size as x when we are supposed to find a scalar. [file:///C:/Users/Utilisateur/Desktop/PROJET_MLA/WenECCV16.pdf](url)

**Comment by jbschlosser** [comment]
The notation in the paper indicates that both `x_i` and `c_{y_i}` are vectors of length `d`. Subtracting one from the other should indeed result in a vector of length `d`. Unless I'm missing something, it looks like you need to take the squared 2-norm of that result to get a scalar.

**Comment by rabahMoulai** [comment]
@jbschlosser thank you for you help, i really appreciate
